ggml/src/ggml-vulkan/vulkan-shaders/vulkan-shaders-gen.cpp

```diff
@@ -138,14 +144,20 @@ void execute_command(const std::string& command, std::string& stdout_str, std::s
         throw std::runtime_error("Failed to fork process");
     }
 
+    std::vector<char*> argv;
+    for (const std::string& part : command) {
+        argv.push_back(const_cast<char *>(part.c_str()));
```

Comment:
If you remove the consts at line 79 and 148,, I think you can avoid this const_cast.

Reply:
No, `c_str()` always returns a `const char *`, because it's accessing the internal data of the string, which you should never modify directly.